src/tmpfiles/tmpfiles.c

```diff
@@ -1810,7 +1826,7 @@ static int item_do(Item *i, int fd, fdaction_t action) {
 
         /* This returns the first error we run into, but nevertheless
          * tries to go on */
-        r = action(i, fd, &st);
+        r = action(i, fd, NULL, &st);
```

Comment:
why don't you pass `path` parameter instead of NULL here ?

Reply:
because i missed it ;-)